maquetado/scss/login.scss

```diff
@@ -5,11 +5,25 @@
   justify-content: center;
 }
 
+@keyframes rotateLogoWolox {
```

Comment:
Just a tiny detail, but I would write `rotation` instead. This animation is not that specific and could be re used further down the road.